sunpy/map/sources/psp.py

```diff
@@ -20,17 +23,43 @@ class WISPRMap(GenericMap):
     * `WISPR Instrument Page <https://wispr.nrl.navy.mil//>`__
     * `Instrument Paper <https://doi.org/10.1007/s11214-014-0114-y>`__
     """
+
+    def __init__(self, data, header, **kwargs):
+        super().__init__(data, header, **kwargs)
+
+        self.plot_settings['norm'] = ImageNormalize(
+            stretch=AsinhStretch(a=0.001), clip=True,
+            vmin=0)
+
     @property
     def processing_level(self):
         lvl = self.meta.get('level', None)
         if lvl is None:
             return
-        return int(lvl[1])
+        # Chop off the leading 'L'
+        lvl = lvl[1:]
+        try:
+            return int(lvl)
+        except ValueError:
+            # The int conversion will fail for L2b files, and we should fail
+            # safe if the user chooses to customize this with other values.
+            return lvl
```

Comment:
```suggestion
        try:
            lvl = int(lvl)
        except ValueError:
            # The int conversion will fail for L2b files, and we should fail
            # safe if the user chooses to customize this with other values.
            pass
        return lvl
```
I'm probably just being crotchety, but I don't like seeing the `return` inside a `try...except` here because the `return` will never raise an exception.